playbooks/azure/openshift-cluster/build_base_image.yml

```diff
@@ -0,0 +1,76 @@
+---
+# Note: this required ansible-galaxy install Azure.azure_modules
+
+- import_playbook: prerequisites.yml
+
+# create all the items necessary to make the VM
+- name: "Create resource group"
+  hosts: localhost
+  gather_facts: no
+  tasks:
+  - name: "provision public ip address"
+    import_role:
+      name: openshift_azure
+      tasks_from: provision_instance.yml
+    vars:
+      openshift_azure_vm_base_image:
+        name: '{{ openshift_azure_root_image }}'
+        resource_group: '{{ openshift_azure_root_image_ns }}'
+
+- name: update OS and install packages
+  hosts: azure
+  gather_facts: yes
+  remote_user: "{{ openshift_azure_vm_admin_username }}"
+  become: True
+  tasks:
+  - name: lay down yum certs
+    copy:
+      content: "{{ item.content }}"
+      dest: "/var/lib/yum/{{ item.name }}"
+    with_items:
+    - name: client-cert.pem
+      content: "{{ yum_client_cert_contents }}"
+    - name: client-key.pem
+      content: "{{ yum_client_key_contents }}"
```

Comment:
setting mode 0600 on this might be a useful additional protection

Reply:
done.